setup/nextcloud.sh

```diff
@@ -2,7 +2,7 @@
 # Nextcloud
 ##########################
 
-source setup/functions.sh # load our functions
+source ../setup/functions.sh # load our functions
```

Comment:
This breaks when running the script from the mailinabox directory

Reply:
Ok

Reply:
Makes sense, just for me I was running it from the setup dir